config.toml.dist

```diff
@@ -173,6 +180,73 @@ session_ttl = 900 # 15 minutes
 # Minimum cache time for downstream HTTP requests made by the broker
 cache_ttl = 3600 # 1 hour
 
+################################################################
+# Rate limits
+
+# Limits can be set in a flexible way. We've tried to provide some sensible
+# defaults, but if you're exposing your broker to the internet, you may need
+# control over limits when the need arises. This section details the available
```

Comment:
I will mention the minor and subjective stuff. 

_when the need arises_ could be left out.